packages/upgrade/src/lib/ast/modifiers.ts

```diff
@@ -16,13 +16,13 @@ export function replaceMemberExpressionRootIndentifier(
   j: JSCodeshift,
   expr: MemberExpression,
   id: Identifier,
-) {
+): boolean {
   if (j.Identifier.check(expr.object)) {
     expr.object = id;
     return true;
   }
-  if (j.MemberExpression.check(expr.object)) {
-    return replaceMemberExpressionRootIndentifier(j, expr.object, id);
-  }
-  return false;
+
+  return !j.MemberExpression.check(expr.object)
+    ? false
+    : replaceMemberExpressionRootIndentifier(j, expr.object, id);
```

Comment:
not sure if this ternary made it more readable but fine

Reply:
There's not a lot of lines in the utils so should be okay, but do let me know if you need it to be more readable